### DISCUSSION TITLE
(Fixed) pacman -Syu issues (as of 20211130)

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#671] [General]
`pacman -Syu` issues: _(as of 20211130)_
- `linux-rpi` 5.10.82 >  `/boot/cmdline.txt` and `/boot/config.txt` - renamed
- `icu` 70.1 > `chromium` 95.0.4638.69 - failed
- `chromium` 96.0.4664.45 > Browser on RPi - failed

Unless you know what to do, don't do it until they're fixed.

**Comment by apastuszak** [comment]
Is this a fix needed in Arch or in Raudio?

**Comment by rern** [comment]
Fixed with today's update

**Important:** 
rAudio with image files prior to `i20211203`,  the renaming of `/boot/cmdline.txt` and `/boot/config.txt` must be fixed with:
```
mv -f /boot/cmdline.txt{.pacsave,}
mv -f /boot/config.txt{.pacsave,}
```
